### PR TITLE
fix(tocco-ui): fix phoneformatter async bug

### PR DESCRIPTION
**Pull request by khanlifts**
PhoneFormatter could get empty object and threw an
error on async loading

### CODE REVIEW

File: packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ describe('tocco-ui', () => {
             .then(() => expect(wrapper.text()).to.equal('+41 44 388 60 00'))
         })
 
-        test('should show original string if its not a valid phonenumber', () => {
-          const invalidPhoneNumber = '+123 456'
+        test('should show original string if its not a valid phone number', () => {
+          const invalidPhoneNumber = '+41 4438860011111110'
```
Comment (MethodenMann) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js]: just curious , what's the motivation behind this change?

File: packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.js [hunk 2]
```diff
@@ -16,9 +16,11 @@ class PhoneFormatter extends React.Component {
...
   render() {
     const libPhoneImport = this.state.libPhoneImport
-    const parsed = libPhoneImport ? libPhoneImport.parseNumber(this.props.value) : this.props.value
-    const formattedNumber = libPhoneImport ? libPhoneImport.formatNumber(parsed, 'International') : parsed
-    return (<span>{_isEmpty(parsed) ? parsed : formattedNumber}</span>)
+    const parsed = libPhoneImport ? libPhoneImport.parseNumber(this.props.value, {extended: true}) : this.props.value
+    const validityParsed = parsed.valid ? parsed : this.props.value
+    const notEmptyValue = _isEmpty(validityParsed) ? this.props.value : validityParsed
+    const formattedNumber = libPhoneImport ? libPhoneImport.formatNumber(notEmptyValue, 'International') : notEmptyValue
+    return (<span>{formattedNumber}</span>)
```
Comment (MethodenMann) [inline packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.js]: You check 2 times if libPhoneImport is defined. Does it have to be that complicated? Would the old version work with a simple check?

something like:
```
if (libPhoneImport) {
const parsed = libPhoneImport.parseNumber(props.value)
return <span>{_isEmpty(parsed) ? props.value : libPhoneImport.formatNumber(parsed, 'International')}</span>
} ellse {
 return <span>{props.value}</span>
}
```

you can also move it to a function

`return <span>{libPhoneImport ? formatWithLib(...) : props.value}</span>`

File: packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js (hunk 1 above)
Comment (khanlifts) [reply packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js]: The library formats the first 2-3 signs as prefix for all the possible areas (+1, +31, +3).

Comment (MethodenMann) [reply packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.spec.js]: As far as i know both are invalid phone numbers. the effect is the same, the component shows the original value unformatted?
